Alter repository properties (#19388)
crate/crate: docs/sql/statements/alter-repository.rst

.. highlight:: psql
.. _sql-alter-repository:

=================
``ALTER REPOSITORY``
=================

Alter the settings of an existing repository.

Synopsis
========

::

    ALTER REPOSITORY name SET ( parameter = value [, ...] )

Description
===========

``ALTER REPOSITORY SET`` changes properties on an existing repository. The type
of the repository cannot be changed.

Executing ``ALTER REPOSITORY`` while a snapshot is being created, restored or

> deleted for the given repository, fails with ``trying to modify or unregister repository that is currently used``.

Reviewer: I wouldn't include the exact error message, because if we change it, the docs will become out of sync. instead I would phrase it as:

```suggestion
deleted for the given repository, fails with an error message.
```
